### ISSUE TITLE
Getting this function takes 3 arguments but 2 arguments were supplied From ModuleEvaluate function

### ISSUE THREAD

**Issue by wolfpat01** [Redfire75369/spiderfire#2]
here is the logs
    ```
   Fresh cc v1.0.67
       Fresh version_check v0.9.3
       Fresh glob v0.3.0
       Fresh unicode-xid v0.2.1
       Fresh autocfg v1.0.1
       Fresh pkg-config v0.3.19
       Fresh regex-syntax v0.6.23
       Fresh rustc-hash v1.1.0
       Fresh shlex v0.1.1
       Fresh lazy_static v1.4.0
       Fresh cfg-if v1.0.0
       Fresh same-file v1.0.6
       Fresh lazycell v1.3.0
       Fresh cfg-if v0.1.10
       Fresh peeking_take_while v0.1.2
       Fresh unicode-width v0.1.8
       Fresh ansi_term v0.11.0
       Fresh strsim v0.8.0
       Fresh vec_map v0.8.2
       Fresh once_cell v1.7.2
       Fresh termcolor v1.1.2
       Fresh regex v1.4.5
       Fresh walkdir v2.3.1
       Fresh textwrap v0.11.0
       Fresh libc v0.2.89
       Fresh memchr v2.3.4
       Fresh proc-macro2 v1.0.24
       Fresh bitflags v1.2.1
       Fresh encoding_rs v0.8.28
       Fresh which v3.1.1
       Fresh log v0.4.14
       Fresh atty v0.2.14
       Fresh time v0.1.44
       Fresh nom v5.1.2
       Fresh libloading v0.5.2
       Fresh quote v1.0.9
       Fresh num-traits v0.2.14
       Fresh libz-sys v1.1.2
       Fresh encoding_c v0.9.8
       Fresh encoding_c_mem v0.2.6
       Fresh clap v2.33.3
       Fresh cexpr v0.4.0
       Fresh clang-sys v0.29.3
       Fresh num-integer v0.1.44
       Fresh bindgen v0.53.3
       Fresh chrono v0.4.19
       Fresh mozjs_sys v0.68.2 (https://github.com/servo/mozjs?rev=82da136c53d7ca7b0b7fe1c5622627036ef31899#82da136c)
       Fresh mozjs v0.14.1 (https://github.com/servo/rust-mozjs#fe104a26)
   Compiling spiderfire v0.1.0 (/home/wolfpat/spiderfire)
     Running `rustc --crate-name spiderfire --edition=2018 src/main.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -Cembed-bitcode=no -C debuginfo=2 -C metadata=f681912fcf3e37c1 -C extra-filename=-f681912fcf3e37c1 --out-dir /home/wolfpat/spiderfire/target/debug/deps -C incremental=/home/wolfpat/spiderfire/target/debug/incremental -L dependency=/home/wolfpat/spiderfire/target/debug/deps --extern chrono=/home/wolfpat/spiderfire/target/debug/deps/libchrono-33f7dbbd138754ae.rlib --extern clap=/home/wolfpat/spiderfire/target/debug/deps/libclap-02d3490164b7a98c.rlib --extern libc=/home/wolfpat/spiderfire/target/debug/deps/liblibc-92a6ce5a991a67c9.rlib --extern mozjs=/home/wolfpat/spiderfire/target/debug/deps/libmozjs-b5ab7020a62ab304.rlib --extern once_cell=/home/wolfpat/spiderfire/target/debug/deps/libonce_cell-e9da1314ab8fbd82.rlib --extern termcolor=/home/wolfpat/spiderfire/target/debug/deps/libtermcolor-e268ff7bc76f4625.rlib -L native=/home/wolfpat/spiderfire/target/debug/build/mozjs-3ca1e6668fc070d0/out -L native=/home/wolfpat/spiderfire/target/debug/build/mozjs_sys-36dbba0e74ff8ac2/out/build/js/src/build -L native=/home/wolfpat/spiderfire/target/debug/build/mozjs_sys-36dbba0e74ff8ac2/out/build/glue`
error[E0061]: this function takes 3 arguments but 2 arguments were supplied
   --> src/runtime/jsapi_utils/eval.rs:103:8
    |
103 |             if !ModuleEvaluate(rt.cx(), module.handle().into()) {
    |                 ^^^^^^^^^^^^^^ -------  ---------------------- supplied 2 arguments
    |                 |
    |                 expected 3 arguments

error: aborting due to previous error

For more information about this error, try `rustc --explain E0061`.
error: could not compile `spiderfire`.

Caused by:
  process didn't exit successfully: `rustc --crate-name spiderfire --edition=2018 src/main.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -Cembed-bitcode=no -C debuginfo=2 -C metadata=f681912fcf3e37c1 -C extra-filename=-f681912fcf3e37c1 --out-dir /home/wolfpat/spiderfire/target/debug/deps -C incremental=/home/wolfpat/spiderfire/target/debug/incremental -L dependency=/home/wolfpat/spiderfire/target/debug/deps --extern chrono=/home/wolfpat/spiderfire/target/debug/deps/libchrono-33f7dbbd138754ae.rlib --extern clap=/home/wolfpat/spiderfire/target/debug/deps/libclap-02d3490164b7a98c.rlib --extern libc=/home/wolfpat/spiderfire/target/debug/deps/liblibc-92a6ce5a991a67c9.rlib --extern mozjs=/home/wolfpat/spiderfire/target/debug/deps/libmozjs-b5ab7020a62ab304.rlib --extern once_cell=/home/wolfpat/spiderfire/target/debug/deps/libonce_cell-e9da1314ab8fbd82.rlib --extern termcolor=/home/wolfpat/spiderfire/target/debug/deps/libtermcolor-e268ff7bc76f4625.rlib -L native=/home/wolfpat/spiderfire/target/debug/build/mozjs-3ca1e6668fc070d0/out -L native=/home/wolfpat/spiderfire/target/debug/build/mozjs_sys-36dbba0e74ff8ac2/out/build/js/src/build -L native=/home/wolfpat/spiderfire/target/debug/build/mozjs_sys-36dbba0e74ff8ac2/out/build/glue` (exit code: 1)
```

**Comment by wolfpat01** [comment]
i forgot to mention that
i'm on windows wsl2
and the files are on `/home/wolfapt/spiderfire`

**Comment by sagudev** [comment]
Fixed in #3
